content/en/docs/ambient/install/multicluster/before-you-begin/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Before you begin
+description: Initial steps before installing Istio on multiple clusters.
+weight: 1
+keywords: [kubernetes,multicluster]
+test: n/a
+owner: istio/wg-environments-maintainers
+---
+
+{{< boilerplate alpha >}}
+
+Before you begin a multicluster installation, review the
+[deployment models guide](/docs/ops/deployment/deployment-models)
+which describes the foundational concepts used throughout this guide.
```

Comment:
Will sending people to the sidecar version of the doc confuse them?

Reply:
IMO, the link isn't a sidecar specific doc; in theory, ambient can support all of the deployment models listed there.